policyengine_uk/parameters/gov/hmrc/tax-free-childcare/childcare_income_condition.yaml

```diff
@@ -0,0 +1,33 @@
+- name: Income requirements and calculations
+  period: 2025
+  absolute_error_margin: 0
+  input:
+    people:
+      person:
+        age: age
+        conditions:
+          - age >= 21 AND (
+              yearly_eligible_income / 4 >= 2379 OR
+              (is_newly_self_employed AND business_age_months < 12)
+            )
+          - (age >= 18 AND age <= 20) AND (
+              yearly_eligible_income / 4 >= 1788 OR
+              (is_newly_self_employed AND business_age_months < 12)
+            )
+          - (age < 18 OR is_apprentice) AND (
+              yearly_eligible_income / 4 >= 1331 OR
+              (is_newly_self_employed AND business_age_months < 12)
+            )
```

Comment:
Are all of these income and age parameters part of the model? We usually test just direct inputs in the unit tests and keep the parameters constant